stacks/core/README.md

```diff
@@ -62,9 +94,9 @@ This stack will be deployed with the basic services, which are related to the th
 
 Firstly, create a new secret to the authenticaton service.
 
-  ```bash
-  openssl rand -base64 256
-  ```
+```bash
+openssl rand -base64 256
+```
```

Comment:
Nit: What's the reasoning behind this change, taking into account that most of the other bash codeblocks are also indented?

Reply:
I didn't notice that change. I think it appeared because I've run the prettier formatter in my editor. BTW, it just removes an unnecessary space.